modules/gitops-release-notes-1-13-0.adoc

```diff
@@ -0,0 +1,128 @@
+   disableMetrics: true
+----
++
+Use the following command to scrape metrics in the Argo CD CLI:
++
+[source,terminal]
+----
+$ oc patch argocd argocd-sample -n default --type='json' -p='[{"op": "replace", "path": "/spec/monitoring/disableMetrics", "value": true}]'
+----
+
+* With this update, the Argo Rollouts functionality in {gitops-title} is promoted from Technology Preview (TP) to General Availability (GA). link:https://issues.redhat.com/browse/GITOPS-3848[GITOPS-3848]
++
+Use this feature to perform the following actions:
++
+** Run cluster-scoped Argo Rollouts instead of namespace-scoped Argo Rollouts
+** Route traffic by using OpenShift Routes for canary application deployment 
+** Configure the Argo Rollouts CLI
+
+* With this update, you can perform canary-based Argo Rollouts deployments by using OpenShift Routes-based traffic management. Argo Rollouts supports traffic splitting by modifying the percentage of traffic that reaches the canary or stable application version during a deployment. link:https://issues.redhat.com/browse/GITOPS-2400[GITOPS-2400]
+
+* With this update, the Argo CD applications in non-control plane namespaces feature is promoted from Technology Preview (TP) to General Availability (GA). link:https://issues.redhat.com/browse/GITOPS-3796[GITOPS-3796]
++
+* With this update, the following new commands are introduced for the Argo CD applications in non-control plane namespaces feature:
+** `argocd proj add-source-namespace`: Use this command to add source namespaces to the `.spec.sourceNamespaces` field of the `AppProject` CR to allow specific namespaces for application creation.
+** `argocd proj delete-source-namespace`: Use this command to remove source namespaces from the `.spec.sourceNamespaces` field of the `AppProject` CR to exclude specific namespaces. link:https://issues.redhat.com/browse/GITOPS-4045[GITOPS-4045]
+
+* With this update, you can customize permissions for a cluster-scoped Argo CD instance by creating user-defined `ClusterRoles` and `ClusterRoleBindings` for the Argo CD Application Controller and server components. link:https://issues.redhat.com/browse/GITOPS-2614[GITOPS-2614]
+
+* With this update, you can customize the Keycloak route name in the {gitops-title} Operator. This update also enables support for complex routing scenarios, such as dividing routes among various ingress controller shards. link:https://issues.redhat.com/browse/GITOPS-4451[GITOPS-4451]
+
+* With this update, the route TLS termination is set as default to the reencrypt mode for both the default and user-defined Argo CD instances. TLS connections to the Argo CD instances now receive the default ingress certificate that is set in OpenShift Container Platform, instead of the self-signed Argo CD certificate generated by the {gitops-title} Operator. You can modify the route TLS termination policy by configuring the ``.spec.server.route.tls` field of the Argo CD CR. link:https://issues.redhat.com/browse/GITOPS-3918[GITOPS-3918]
```

Comment:
```suggestion
* With this update, the route TLS termination is set as default to the reencrypt mode for both the default and user-defined Argo CD instances. TLS connections to the Argo CD instances now receive the default ingress certificate that is set in OpenShift Container Platform, instead of the self-signed Argo CD certificate generated by the {gitops-title} Operator. You can modify the route TLS termination policy by configuring the `.spec.server.route.tls` field of the Argo CD CR. link:https://issues.redhat.com/browse/GITOPS-3918[GITOPS-3918]
```

Gives a rendering issue also: 
![image](https://github.com/openshift/openshift-docs/assets/77115114/f7185e87-1354-4ec2-9bb9-be81f93f5ff8)

Reply:
Fixed.